ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivCall.java

```diff
@@ -60,9 +60,6 @@ protected Object resultByBlockNumber(
 
     final String privacyUserId = privacyIdProvider.getPrivacyUserId(request.getUser());
 
-    PrivUtil.checkMembershipForAuthenticatedUser(
```

Comment:
Where is this check happening now?

Reply:
At the moment that check is supposed to happen in the PrivacyController. We only have to do that check if we are doing multi-tenancy. In that case the MultiTenancyPrivacyController will take care of that, in the simulatePrivateTransaction method.